### ISSUE TITLE
"run code in-line" button is failing with a python error

### ISSUE THREAD

**Issue by DavidHowlett** [Komodo/KomodoEdit#810]
I am using:
    komodo 9.3
    windows 7
    python 3.4 (anaconda package)

My program is:
    print("hello world")

When I run it in the debugger it works fine (see screenshot). When I select all the text and press the lightning button I get the below error message:
    Fatal Python error: Failed to initialize Windows random API (CryptoGen)

    FAIL: 3

Screenshot of error message:
    https://www.dropbox.com/s/xujb3dwrlsd6irp/Screenshot%202015-11-05%2015.50.25.png?dl=0

**Comment by Naatan** [comment]
Hmm this working fine for me on Linux. Possibly this is Windows or Anacona specific. Most likely you need to be exposing an ENV var.

The real error of importance here is "Fatal Python error: Failed to initialize Windows random API (CryptoGen)".

This seems relevant: https://bugs.python.org/issue20614

Try setting SYSTEMROOT in Prefs > Environment. to find out what the value is supposed to be open a command prompt and execute `echo %SYSTEMROOT%`, the output is the value of SYSTEMROOT. Most likely this is `C:\WINDOWS\`.

**Comment by DavidHowlett** [comment]
when I type: echo %SYSTEMROOT% 
I get: C:\Windows

When I look at the Environment variables 
I get: C:\Windows

see screenshot:
https://www.dropbox.com/s/3k4g9orggmd64ii/Screenshot%202015-11-05%2016.12.13.png?dl=0

**Comment by Naatan** [comment]
Ok .. so did it work?

**Comment by DavidHowlett** [comment]
Ahh, I did not communicate well. I did not change %SYSTEMROOT%  or the Environment variables. They started out the same. They are still the same.

I tried upgrading from from python 3.4 to 3.5
The error now looks like:

    FAIL: 3221225781

The screenshot:
https://www.dropbox.com/s/a3shast2so9fslo/Screenshot%202015-11-05%2017.21.38.png?dl=0

If you want I will try switching away from anaconda python to the standard distribution and see if that fixes the issue.
